### PR TITLE
Fix for daily interval trigger and Cron serialization

### PR DESCRIPTION
**Pull request by jenrom**
You will find multiple test cases that explain the issues. I hope it will help

### CODE REVIEW

**Comment by Nanoko** [comment]
Your serializers look good. Could you please put them in the package namespace just like other serializers ?

File: src/Quartz.Impl.MongoDB/TimeZoneInfoSerializer.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+using MongoDB.Bson.IO;
+using MongoDB.Bson.Serialization;
+
+namespace AutoScout24.SMP.BackgroundServices.SmpJobs.Infrastructure.Quartz
```
Comment (Nanoko) [inline src/Quartz.Impl.MongoDB/TimeZoneInfoSerializer.cs]: Replace with : namespace Quartz.Impl.MongoDB

File: src/Quartz.Impl.MongoDB/TimeOfDaySerializer.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+using MongoDB.Bson;
+using MongoDB.Bson.IO;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Options;
+
+using Quartz;
+
+namespace AutoScout24.SMP.BackgroundServices.SmpJobs.Infrastructure.Quartz
```
Comment (Nanoko) [inline src/Quartz.Impl.MongoDB/TimeOfDaySerializer.cs]: Replace with : namespace Quartz.Impl.MongoDB

File: src/Quartz.Impl.MongoDB/JobStore.cs [hunk 3]
```diff
@@ -27,6 +27,8 @@
 using System.Text;
 using System.Threading;
 
+using AutoScout24.SMP.BackgroundServices.SmpJobs.Infrastructure.Quartz;
```
Comment (Nanoko) [inline src/Quartz.Impl.MongoDB/JobStore.cs:30]: Remove this line

**Comment by jenrom** [comment]
I've fixed that. You can just pull the changes

**Pull request merged**